README.md

```diff
@@ -112,6 +112,13 @@ the time of the next major release; therefore we try to keep each RFC
 document somewhat in sync with the language feature as planned,
 tracking such changes via followup pull requests to the document.
 
+### RFC Statuses
+
+1. DRAFTED - RFC is actively in development and seeking feedback.
+2. ACTIVE - RFC is ready for implementation.
+3. APPROVED - RFC is being implemented.
+4. IMPLEMENTED - Implementation has been completed.

```

Comment:
I thought it would be helpful to add definitions to each status as I wasn't sure what each status meant/how the status progression worked.  

For example, does ACTIVE come before APPROVED?  

I'm also not sure what `APPROVED` means but based on how it's using in the previous RFCs it is either:

1. The RFC is actively being implemented.
2. The same thing as IMPLEMENTED.

Would love to get some feedback on this section and open to changing the wording on any of these.

Reply:
We based our implementation on [Ember's](https://github.com/emberjs/rfcs#stages), looks like they're not quite the same anymore (if they ever were).

I would remove ACTIVE entirely because I'm not sure there's a distinction between ACTIVE and DRAFTED.

Reply:
I'd say APPROVED means "The RFC is actively being implemented"